Marlin/ultralcd.cpp

```diff
@@ -1359,7 +1359,7 @@ void kill_screen(const char* lcd_msg) {
   static void lcd_move_z() { _lcd_move_xyz(PSTR(MSG_MOVE_Z), Z_AXIS, sw_endstop_min[Z_AXIS], sw_endstop_max[Z_AXIS]); }
   static void _lcd_move_e(
     #if E_MANUAL > 1
-      int8_t eindex
+      int8_t eindex=-1
     #endif
```

Comment:
There's no argument at all if `E_MANUAL` is 1, since only a single extruder can move. With `E_MANUAL > 1` the call to `_lcd_move_e()` moves the active extruder, otherwise `_lcd_move_e(n)` moves a specific extruder.